tests/plugins/range.test.ts

```diff
@@ -124,6 +124,62 @@ describe("range plugin", () => {
       });
     });
 
+    describe("create a range and remove multiple columns", () => {
+      beforeEach(() => {
+        m = new Model({
+          sheets: [
+            { id: "s1", name: "s1" },
+            { id: "s2", name: "s 2" },
+          ],
+        });
+        m.dispatch("USE_RANGE", { sheetId: m.getters.getActiveSheetId(), rangesXC: ["C2:F5"] });
+      });
+
+      test("in the middle", () => {
+        deleteColumns(m, ["D", "E"]);
+        expect(m.getters.getUsedRanges()).toEqual(["C2:D5"]);
+      });
+
+      test("in the start", () => {
+        deleteColumns(m, ["B", "C"]);
+        expect(m.getters.getUsedRanges()).toEqual(["B2:D5"]);
+      });
+
+      test("in the end", () => {
+        deleteColumns(m, ["E", "F"]);
+        expect(m.getters.getUsedRanges()).toEqual(["C2:D5"]);
+      });
+
+      test("before the start", () => {
+        deleteColumns(m, ["A", "B"]);
+        expect(m.getters.getUsedRanges()).toEqual(["A2:D5"]);
+      });
+
+      test("after the end", () => {
+        deleteColumns(m, ["G", "H"]);
+        expect(m.getters.getUsedRanges()).toEqual(["C2:F5"]);
+      });
+
+      test("including one column before the start and the first column", () => {
+        deleteColumns(m, ["C", "B"]);
+        expect(m.getters.getUsedRanges()).toEqual(["B2:D5"]);
+      });
+
+      test("including one column after the end and the last column", () => {
+        deleteColumns(m, ["G", "F"]);
+        expect(m.getters.getUsedRanges()).toEqual(["C2:E5"]);
+      });
+
+      test("all columns in the sheet", () => {
+        const numberOfCols = m.getters.getNumberCols(m.getters.getActiveSheetId());
+        deleteColumns(
```

Comment:
you cannot delete all the columns of the sheet, this command is rejected so it will never actually be processed by `RangeAdapter` nor `PluginTestRange`, I might be missing the point of that test ?

Reply:
@rrahir I knew this behavior so it is here to make the test range more complete. Should I just add a comment here or delete this (and similar) tests?